Add a `Create a DANDI Account` page (#116)
dandi/handbook: docs/16_account.md

# Create a DANDI Account

A DANDI account is only required for specific features.
Without a DANDI account, you can search, view, and download files.
With a DANDI account, you can additionally create Dandisets and access the DANDI Hub to analyze existing data.

## Instructions

1. To create a DANDI account, first [create a GitHub account](https://github.com/) if you don't have one.
1. Using your GitHub account, register for a DANDI account by selecting the `LOG IN WITH GITHUB` button on the [DANDI homepage](https://dandiarchive.org).

> 1. You will receive an email acknowledging that your request for an account will be reviewed within 24 hours:

Author: ```suggestion
1. You will receive an email acknowledging that your request for an account will be reviewed within 24 hours.
```

> 1. You will receive an email acknowledging activation of your account within 24
> hours, after which you can log in to DANDI using GitHub by clicking the 
> `LOG IN WITH GITHUB` button.

Reviewer: for consistency and to set expectations:

```suggestion
1. You will receive an email acknowledging that your request for an account will be reviewed within 24 hours:
  - **note**: requests from a new GitHub accounts and from emails not withing `.edu` domain might take longer to review and more likely to be rejected, especially if there are no plans described to upload data to the archive
1. If your request for an account approved, you would be able to log in to DANDI using GitHub by clicking the `LOG IN WITH GITHUB` button.
```

>   - **note**: requests from a new GitHub accounts and from emails not withing `.edu` domain might take longer to review and more likely to be rejected, especially if there are no plans described to upload data to the archive

Author: ```suggestion
  - **note**: Requests from new GitHub accounts and from emails that are not an `.edu` domain might take longer to review and are more likely to be rejected, especially if there are no plans described to upload data to the archive.
```

> 1. If your request for an account approved, you would be able to log in to DANDI using GitHub by clicking the `LOG IN WITH GITHUB` button.

Author: ```suggestion
1. If your request for an account is approved, you will be able to log in to DANDI using GitHub by clicking the `LOG IN WITH GITHUB` button.
```
